crates/ruff_python_parser/src/parser.rs

```diff
@@ -698,9 +699,9 @@ array[3:5, *indexes_to_select]
             r#"try:
     raise ValueError(1)
 except TypeError as e:
-    print(f'caught {type(e)}')
+    print('caught', type(e))
 except OSError as e:
-    print(f'caught {type(e)}')"#,
+    print('caught', type(e))"#,
```

Comment:
The tests are for `try ... except ...` so let's use normal strings to avoid putting the test behind feature flag.